<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打分系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="strTip"/>
     </item>
     <item>
      <widget class="QTableWidget" name="tableClient">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>当前选手号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnOrder">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>当前环节号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="btnHuanjie">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>当前场次号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>当前轮次号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnChangci">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="btnLunci">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinOrder"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinChangci"/>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="spinHuanjie"/>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="spinLunci"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBtn">
     <item>
      <widget class="QPushButton" name="btnInit">
       <property name="text">
        <string>初始化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWelcome">
       <property name="text">
        <string>显示主界面</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCountDown">
       <property name="text">
        <string>倒计时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCountPause">
       <property name="text">
        <string>暂停计时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCountStop">
       <property name="text">
        <string>停止计时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnScore">
       <property name="text">
        <string>开始打分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRank">
       <property name="text">
        <string>显示排名</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
